python-package/mlbox/encoding/categorical_encoder.py

```diff
@@ -9,7 +9,7 @@
 import os
 
 # Set the keras backend if not set, default is theano
-if not "KERAS_BACKEND" in os.environ:
+if "KERAS_BACKEND" not in os.environ:
     os.environ["KERAS_BACKEND"] = "theano"
 
 from keras.layers.core import Dense, Reshape, Dropout
```

Comment:
@NCherel @AxeldeRomblay 

Usually, imports are made at the top of the module (`pep8`standards).
But here we cannot do so because of the `KERAS_BACKEND` that we need to set prior to the imports.

Any idea on how to deal with this `pep8` issue?
`E402 module level import not at top of file`